ext/PlottingExt/PlotSpins.jl

```diff
@@ -81,11 +81,11 @@ function Sunny.plot_spins!(ax, sys::System; notifier=Makie.Observable(nothing),
 
     # Parameters defining arrow shape
     a0 = arrowscale * ℓ0
-    arrowsize = 0.4a0
-    linewidth = 0.12a0
-    lengthscale = 0.6a0
-    markersize = 0.8linewidth
-    arrow_fractional_shift = 0.6
+    shaftradius = 0.06a0
+    tipradius = 0.2a0
+    tiplength = 0.4a0
+    lengthscale = 1.0a0
+    markersize = 1.6shaftradius
```

Comment:
These mappings are exact:
```jl
tipradius = arrowsize/2
shaftradius = linewidth/2
tiplength = arrowsize
```

The new total `lengthscale` matches the old `lengthscale` (previously only for the shaft) plus the `tiplength`.